.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         dbg: [OFF, ON]
-        vm: [ANONYMOUS, SHM]
+        vm: [ANONYMOUS, SHM, MEMFD]
```

Comment:
Really minor: perhaps `MEMFD` should be placed before `SHM` to preserve alphabetical order.